pkg/routes/imagesets.go

```diff
@@ -115,7 +115,7 @@ func ListAllImageSets(w http.ResponseWriter, r *http.Request) {
 		json.NewEncoder(w).Encode(&countErr)
 		return
 	}
-	fmt.Printf("r.URL.Query() %v \n", r.URL.Query().Get("sort_by"))
+	log.Errorf("r.URL.Query() %v \n", r.URL.Query().Get("sort_by"))
```

Comment:
```suggestion
	log.Debugf("r.URL.Query() %v \n", r.URL.Query().Get("sort_by"))
```

maybe? doesn't look like an error or something relevant enough for Info level